### DISCUSSION TITLE
antd 组件部分功能因开发环境访问地址不一致导致有些功能不能正常使用

### DISCUSSION THREAD

**Discussion by Lin00yi** [ant-design/ant-design#49018] [Q&A]
react18 不论是vite构建还是createApp构建，开发环境使用localhost访问 ，colorPicker、select等组件使用不了，只展示默认样式；在局域网下访问就能正常使用该组件（图一localhost；图二局域网）
![image](https://github.com/ant-design/ant-design/assets/103816112/c386d966-e7f3-4649-bf7d-fc160d8cfdf5)
![image](https://github.com/ant-design/ant-design/assets/103816112/46e2c275-c5d4-4ca7-b6af-ecd38cd1219c)

**Comment by Wxh16144** [comment]
和 UI 框架问题不大吧？可能是一些网络问题导致资源加载错误，最好提供一个简单的复现仓库协助我们排查问题。

**Reply by Lin00yi** [reply]
网络问题不应该吧 我重新根据antd官方 create-react-app和vite分别创建了最基础的 还是一样的问题

**Reply by Wxh16144** [reply]
> 重新根据antd官方 create-react-app和vite分别创建了最基础的 还是一样的问题

那你就把这个 repo push 上来， 需要知道你使用的 vite 、antd 的版本信息和本地环境信息:

```bash
npx envinfo --system --npmPackages vite,antd,react --binaries --browsers
```

**Reply by Lin00yi** [reply]
https://github.com/Lin00yi/antd-dev-problem.git
envinfo.txt：
  System:
    OS: Windows 11 10.0.22631
    CPU: (12) x64 11th Gen Intel(R) Core(TM) i5-11400 @ 2.60GHz
    Memory: 3.16 GB / 15.80 GB
  Binaries:
    Node: 18.18.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.21 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.7.6 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.22621.3527
